packages/@react-spectrum/card/docs/CardView.mdx

```diff
@@ -0,0 +1,522 @@
+To programmatically control card selection, use the `selectedKeys` prop paired with the `onSelectionChange` callback. The `key` prop from the selected card will
+be passed into the callback when the card is pressed, allowing you to update state accordingly.
+
+Here is how you would control selection for the above example.
+
+```tsx example export=true
+function PokemonCardView(props) {
+
+  let items = [
+    // Same items as in the pokemon examples...
+    ///- begin collapse -///
+    {id: 1, name: 'Bulbasaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/001.png'},
+    {id: 2, name: 'Ivysaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/002.png'},
+    {id: 3, name: 'Venusaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/003.png'},
+    {id: 4, name: 'Charmander', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/004.png'},
+    {id: 5, name: 'Charmeleon', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/005.png'},
+    {id: 6, name: 'Charizard', type: 'Fire, Flying', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/006.png'}
+    ///- end collapse -///
+  ];
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+
+  return (
+    <CardView items={items} height="500px" maxWidth="800px" layout={GridLayout} aria-label="Controlled selection CardView Pokemon example" selectionMode="multiple"selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} {...props}>
+      {item => (
+        <Card textValue={item.name} width={item.width} height={item.height}>
+          <Image src={item.src} />
+          <Heading>{item.name}</Heading>
+          <Text slot="detail">PNG</Text>
+          <Content>{item.type}</Content>
+        </Card>
+      )}
+    </CardView>
+  );
+}
+```
+
+### Single selection
+To limit users to selecting only a single card at a time, `selectionMode` can be set to `single`.
+
+```tsx example
+// Using the same CardView as above
+<PokemonCardView selectionMode="single" />
+```
+
+### Disallow empty selection
+CardView also supports a `disallowEmptySelection` prop which forces the user to have at least one Card in the CardView selected at all times.
+In this mode, if a single card is selected and the user presses it, it will not be deselected.
+
+```tsx example
+// Using the same CardView as above
+<PokemonCardView disallowEmptySelection />
+```
+
+### Disabled cards
+You can disable specific card by providing an array of keys to CardView via the `disabledKeys` prop. This will prevent cards from being selectable as shown in the example below.
+
+```tsx example
+// Using the same CardView as above
+<PokemonCardView selectionMode="multiple" disabledKeys={[3]} />
```

Comment:
Should we hide the checkbox completely when selection is disabled on a particular card rather than just disabling it? We could also disable the hover state if a card isn't interactive at all (either selectable or navigable) like in TableView. A better example might also be to have folders and files, and make only files selectable. Right now it doesn't really make sense that one item isn't selectable.

Reply:
Sounds good, I'll remove the checkbox if selection is disabled on a card. Its the oddest thing though, the hover state should already be disabled if the card is disabled... It works properly in the storybook but not in the docs for some reason.